hikari/audit_logs.py

```diff
@@ -310,6 +445,38 @@ class AuditLogEntry(snowflakes.Unique):
     reason: typing.Optional[str] = attr.ib(eq=False, hash=False, repr=False)
     """The reason for this change, if set (between 0-512 characters)."""
 
+    async def fetch_user(self) -> typing.Optional[users_.User]:
```

Comment:
I was just thinking about this, but would a `fetch_member` be more useful here? Maybe even as another method

Reply:
there is not guild_id though